app/views/dashboard/_pending_applications.haml

```diff
@@ -10,5 +10,6 @@
               %li= link_to record.user.proper_name, record
           - else
             No pending applications
-        = link_to 'View saved applications for ' + position.name,
+        - count = position.saved_applications.count
+        = link_to "View saved applications for #{position.name} (#{count})",
```

Comment:
I agree with having the text "no saved applications." I did not agree with just showing the default text that it has shown for months when there are none. Will make that change.